### PR TITLE
adding compas_libigl

### PR DESCRIPTION
**Pull request by brgcode**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: .azure-pipelines/azure-pipelines-win.yml [hunk 1]
```diff
@@ -62,6 +62,10 @@ jobs:
     - script: conda.exe config --add channels conda-forge
       displayName: configure conda channels
 
+    # Add the cahnnel for gmp & mpfr
+    - script: conda.exe config --add channels salilab
```
Comment (xhochy) [inline .azure-pipelines/azure-pipelines-win.yml:66]: Please also add these packages to conda-forge.

**Comment by brgcode** [comment]
ha ok. sorry will do. had pushed the previous change before i saw you comment. thanks!

File: .azure-pipelines/azure-pipelines-win.yml (hunk 1 above)
Comment (brgcode) [reply .azure-pipelines/azure-pipelines-win.yml]: will do. in the meantime i have removed the windows dependencies i was trying to get from that channel (and the corresponding functionality of the package)

**Comment by isuruf** [comment]
This seems to be vendoring `libigl`.

**Comment by brgcode** [comment]
@isuruf the package is indeed a wrapper around some specific functions of `libigl`. do i have to do something specific in relation to this? please forgive my ignorance. i am a bit new to this process...

**Pull request closed without merging**